test/plugin/test_exception_detector.rb

```diff
@@ -256,6 +256,50 @@ class ExceptionDetectorTest < Test::Unit::TestCase
   at System.Threading.Thread.StartInternal () [0x00000] in <filename unknown>:0
 END
 
+  CSHARP_NESTED_EXC = <<END.freeze
```

Comment:
It might be time to define an array of exceptions per language. This is a much larger refactoring than we should reasonably expect you to do in this PR, but if you're game, that's great. If not, we could try to do it ourselves, and then rebase your PR on top of the result. @qingling128, WDYT?

Reply:
I'm fine with merging this PR once it's approved and refactoring them into arrays per language later. This way we unblock the use case earlier.